src/Tracy/assets/Bar/bar.js

```diff
@@ -357,9 +357,22 @@
 		if (!layer.dataset.id) {
 			return;
 		}
+		var reURLInformation = new RegExp('^(https?:)//(([^:/?#]*)(?::([0-9]+))?)(/[^?#]*)');
+	        var oldOpen = XMLHttpRequest.prototype.open;
+	        XMLHttpRequest.prototype.open = function(){
```

Comment:
what about `this.isCrossDomain= arguments[1].indexOf('//') > -1 && arguments[1].indexOf(window.location.origin) < 0`